src/benchmarks/micro/corefx/System.Linq/Perf.Linq.cs

```diff
@@ -101,13 +103,17 @@ public void OrderByThenBy(int size, int iteration, Perf_LinqTestBase.WrapperType
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.OrderBy(o => -o).ThenBy(o => o), _consumer);
 
         [Benchmark]
-        [Arguments(DefaultSize, DefaulIterationCount)]
```

Comment:
breaking change: parameter names are part of benchmark ID in BenchView. So to keep the BenchView ID we have to keep the old and unused arguments ;( I don't like it myself, we should discuss that with @brianrob and @billwert

Reply:
There are a bunch of places where there are inconsisten things.  I suggest we accept a break on BenchView and do an overall cleanup, which will make us more productive moving forward.  I'm also not particularly concerned about a break in BenchView like this: if you really need to go back further and look at the history, it would just be indexed under another ID, right?  Plus, with the perf repo's support for running against multiple runtimes, I can easily run the benchmark against net472, 2.0, 2.1, and 3.0, even without BenchView historical data.

That said, I'll revert this to keep that a separate discussion.

Reply:
> I suggest we accept a break on BenchView and do an overall cleanup

I agree. When I was porting the benchmarks here I was not sure about this. I am glad you share the same opinion.

> That said, I'll revert this to keep that a separate discussion.

Thank you!